ipatests/test_integration/test_replica_promotion.py

```diff
@@ -568,6 +568,18 @@ def check_pki_error(self, host):
         # check for cert/key import error message
         assert self.ERR_MESS not in pki_debug_log
 
+    def switch_selinux_to_enabled(self, host):
+        """Candidate to move to tasks.py
+        """
+        cmd = ["setenforce", "1"]
+        status = host.run_command(cmd)
+        assert "usage:  setenforce" not in status.stderr_text
```

Comment:
I don't think the assert is needed. If setenforce returns 1 then something went wrong and it will raise an exception in that case.